crypto/fipsmodule/bn/bytes.c

```diff
@@ -143,41 +143,72 @@ BIGNUM *BN_le2bn(const uint8_t *in, size_t len, BIGNUM *ret) {
   // Make sure the top bytes will be zeroed.
   ret->d[num_words - 1] = 0;
 
-  // We only support little-endian platforms, so we can simply memcpy the
-  // internal representation.
+#ifdef OPENSSL_BIG_ENDIAN
+  BN_ULONG word = 0;
+  unsigned m;
+
+  m = (len - 1) % BN_BYTES;
+  for (size_t i = len - 1; i < len; i--) {
```

Comment:
if we're initializing `i = len-1`, and the loop condition is `i < len` as we decrement, then the loop will only terminate when `i` underflows, right? would making the loop condition `i >= 0` be more clear?

Reply:
size_t is unsigned so i >= 0 is always true.

Reply:
Right, that makes sense. And we need to cover `i = 0`, so i guess the underflow is necessary. Perhaps we can add a small comment stating the intention?

Reply:
Adding comment:
```c
  // size_t is unsigned so i >= 0 is always true 
  for (size_t i = len - 1; i < len; i--) {
```